clients/launcher/lib/widgets/welcome_modal.dart

```diff
@@ -110,191 +188,631 @@ class _WelcomeModalContentState extends State<_WelcomeModalContent> {
+                              border: OutlineInputBorder(
+                                borderRadius: BorderRadius.all(Radius.circular(2)),
+                              ),
+                              enabledBorder: OutlineInputBorder(
+                                borderRadius: BorderRadius.all(Radius.circular(2)),
+                                borderSide: BorderSide(color: Colors.white24),
+                              ),
+                              focusedBorder: OutlineInputBorder(
+                                borderRadius: BorderRadius.all(Radius.circular(2)),
+                                borderSide: BorderSide(color: Colors.white),
+                              ),
+                            ),
+                            onChanged: _handleEntropyInput,
+                          ),
+                        ),
+                      ),
+                      const SizedBox(width: 8),
+                      Row(
+                        children: [
+                          SizedBox(
+                            height: 16,
+                            width: 16,
+                            child: Checkbox(
+                              value: _isHexMode,
+                              onChanged: (value) {
+                                setState(() {
+                                  _isHexMode = value ?? false;
+                                  if (_entropyController.text.isNotEmpty) {
+                                    _handleEntropyInput(_entropyController.text);
+                                  }
+                                });
+                              },
+                              side: const BorderSide(color: Colors.white70),
+                              fillColor: WidgetStateProperty.resolveWith<Color>(
+                                (Set<WidgetState> states) {
+                                  if (states.contains(WidgetState.selected)) {
+                                    return Colors.white24;
+                                  }
+                                  return Colors.transparent;
+                                },
+                              ),
+                            ),
+                          ),
+                          const SizedBox(width: 4),
+                          const Text(
+                            'hex',
+                            style: TextStyle(fontSize: 9, color: Colors.white70),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 4),
+                  // BIP39 Info
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      const Text(
+                        'BIP39 Information',
+                        style: TextStyle(fontSize: 9, fontWeight: FontWeight.bold, color: Colors.white),
+                      ),
```

Comment:
Use `SailText.<size that makes sense>`. That will use the correct font, and automatically set the correct color based on the users theme